projects/uitdatabank/docs/entry-api/organizers/linking.md

```diff
@@ -0,0 +1,9 @@
+Coming soon.
+
+<!-- 
+  @todo
+  - Explain who can link which organizers to events/places (= anyone can link any organizer to their own events/places) 
+  - Explain that the linked organizer does NOT give permission to people that work at that organisation to make edits to the event
```

Comment:
... unless there are roles set up, allowing people from city XX to edit all events organized in city XX or other validator roles. This (comment) gives the impression that an API-entered event is protected against editing by others.

Reply:
Yes I can add that, but my note was a reminder to explain that a linked organizer does not automatically grant such permissions by default. Because that is an assumption that some integrators seem to have (based on support questions that I've seen pass by in the past)